### DISCUSSION TITLE
Hiding bookmark sidebar on start up - how

### DISCUSSION THREAD

**Discussion by thomasmstein** [aaFn/Bookmark-search-plus-2#314] [Q&A]
I want the bookmark sidebar hidden on browser start-up by default. I don't seem to find the switch to set it. 
My apologies if I overlooked something in the config section.
Thanks

**Comment by aaFn** [comment]
Hello, that is not something that an add-on can master / control.
Having the sidebar open or not at start up is a Firefox behavior (and I do not know if there is a setting for it).

**Reply by thomasmstein** [reply]
Thank you. I have been using this extension for quite a while and it used to show the same behaviour as the default bookmarks.

**Default behaviour of Firefox** 
_(With Bookmark-search-plus-2 deactivated or not installed)_

If I shutdown Firefox with default bookmark sidebar opened 
-> On restart of Firefox bookmarks sidebar is **_shown_**

If I shutdown Firefox with default bookmark sidebar closed
-> On restart of Firefox bookmarks sidebar are **_NOT shown_**

**With Bookmark-search-plus-2 installed and activated**
When Firefox is being shut down, regardless whether Bookmark-search-plus-2 is open or closed in the sidebar 
-> On restart of Bookmark-search-plus-2 is **_ALWAYS shown in sidebar_**

Thank you.

**Comment by aaFn** [comment]
Yep, you are the second to report that, see https://github.com/aaFn/Bookmark-search-plus-2/issues/312, but this is not something controlled by the add-on.

On my side, the behavior you describe is still happening, so I suggest that you try again but with a blank profile in the same FF where BSP2 is installed - all chances are that there is something in your FF profile which is causing that, and which was not appearing before FF 130, but it is appearing now.

**Comment by aaFn** [comment]
See this https://github.com/aaFn/Bookmark-search-plus-2/issues/316, I believe this will be corrected in FF 133.
